src/stream.rs

```diff
@@ -26,7 +26,6 @@ pub async fn stream_units(
     rpc: MissionRpc,
     tx: Sender<Result<Update, Status>>,
 ) -> Result<(), Error> {
-    // initialize the state for the current units stream instance
```

Comment:
Nit: why remove the comment, is it that useless? 😄

Reply:
My bad. I had a few lines commented and then just deleted all the comments without realising I took something extra with it.